bin/plot_chpi_snr.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Plot signal-to-noise of continuous HPI coils as a function of time.
+Works by fitting a general linear model (HPI freqs, line freqs, DC, slope) to
+the data, and comparing estimated HPI powers with the residual (=variance
+unexplained by the model).
+Window length for SNR estimates can be specified on the command line.
```

Comment:
empty newline between paragraphs otherwise it gets rendered as a single paragraph